index.html

```diff
@@ -0,0 +1,533 @@
+            <span class="small-text">Matthew Beedham 17 hours ago</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/hdc-2020-img-1.jpg" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h4>Everything we expect to see at Huawei’s 2020 dev conference tomorrow</h4>
+            <span class="small-text">Bryan Clark 20 hours ago</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/bicyle.png" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h4>The Juiced HyperScramber 2 ebike goes 100+ miles per charge using a pair of massive batteries</h4>
+            <span class="small-text">Napier Lopez 20 hours ago</span>
+          </div>
+        </article>
+
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/mac.jpg" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h4>Xbox Series X and Series S launching November 10 for $499 and $299, respectively</h4>
+            <span class="small-text">Mix 22 hours ago</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/cartoon.png" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h4>Xbox Game Pass is adding EA Play games to its lineup</h4>
+            <span class="small-text">Tristan Greene 23 hours ago</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/remote-work-home-gq-368x276.png" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h4>You’ll know how to find the valuable data and even make machines think for under $40</h4>
+            <span class="small-text">TNW Deals 1 day ago</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/trump.png" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h4>This AI crunched the numbers to make the perfect TikTok song — but is it though?</h4>
+            <span class="small-text">Thomas Macaulay 1 day ago</span>
+          </div>
+        </article>
+        <article class="latest-news">
+          <div class="col-1">
+            <img src="./images/stonksgates.jpg" alt="" class="main-img">
+          </div>
+          <div class="col-2">
+            <h4>This AI crunched the numbers to make the perfect TikTok song — but is it though?</h4>
+            <span class="small-text">Thomas Macaulay 1 day ago</span>
+          </div>
+        </article>
+      </div>
+    </section>
+    <div class="flex">
```

Comment:
- [ ] There is a redundant empty space on the right side of your page in the medium screen view. Please fix your content to fill the full width independent from the screen size.

![image](https://user-images.githubusercontent.com/60184961/93029122-8e4d3680-f621-11ea-95a4-f242c3c69f7f.png)